public/index.html

```diff
@@ -45,6 +45,6 @@
         React.createElement(Reviews.default),
         document.getElementById('reviews')
       );
-    </script>
+    </script> -->
   </body>
 </html>
```

Comment:
per AirBnB style guide there should be an extra line at end of file (didn't have anything else to say)